Libraries/DirectMLX.h

```diff
@@ -3304,6 +3304,7 @@ namespace dml
         Expression input,
         TensorDimensions outputSizes,
         DML_INTERPOLATION_MODE mode,
+        DML_AXIS_DIRECTION roundingDirection,
```

Comment:
Adding this will break the API for `DML_RESAMPLE1_OPERATOR_DESC`

Reply:
If we keep DMLX current with the recent DML bits, then moving it forward is the right thing to do. Asked Adrian for clarification on level of DMLX compat guarantees...

Reply:
To lessen the breaking caller impact, we could set it to a default parameter.

```
        DML_INTERPOLATION_MODE mode = DML_INTERPOLATION_MODE_LINEAR,
        DML_AXIS_DIRECTION roundingDirection = DML_AXIS_DIRECTION_DECREASING,
        ...
```